apps/frontend/src/libs/enums/notification-message.enum.ts

```diff
@@ -1,4 +1,5 @@
 const NotificationMessage = {
+	GROUP_CREATE_SUCCESS: "Group was successfully created",
```

Comment:
End messages with a period
```suggestion
	GROUP_CREATE_SUCCESS: "Group was successfully created.",
```